README.md

```diff
@@ -6,29 +6,49 @@ Paper: http://arxiv.org/abs/2208.01066 <br><br>
 
 
 ## Getting started
-You can start by cloning the repo and following the steps below.
 
-1. Install the dependencies for our code using Conda. You may need to adjust the environment YAML file depending on your setup. *(we did not)*
+<details>
+    <summary><h3>Local Environment</h3></summary>
+To get started with this codebase:
+
+1) Clone the repo  
+
+2) Install the dependencies for our code using Conda. You may need to adjust the environment YAML file depending on your setup (alternatively, you can use a Codespace (described below)).  
 
     ```
     conda env create -f environment.yml
     conda activate in-context-learning
     ```
+</details>
+
+<details>
+<summary><h3>Codespaces (new!)</h3></summary>
+Click <a href="https://github.com/codespaces/new?hide_repo_select=true&ref=devcontainer-improvement&repo=724878312">this link</a> and step through the configurator or start a <a href="https://github.com/features/codespaces">codespace</a> directly from this repo. That's it -- dependencies will be automatically installed and you will be dropped in your default codespace editor.
+
+Note: If the conda environment is not automatically activated, you may need to run 
+
+    ```
+    source /opt/conda/bin/activate in-context-learning
+    ```
+</details>
+
+### Training
 
-2. Start training with:
+You can start training with:
 
     ```
     cd src/
-    python train.py --config conf/any-task_relu_attn.yml
+    python train.py --config conf/experiments/any-task_relu.yml
```

Comment:
should be .yaml, not .yml. and also "FileNotFoundError: [Errno 2] No such file or directory: 'conf/any-task_relu_attn.yaml'".

The only files I have under /conf are:
base_curriculum.yaml		experiments			tasks				toy_cloud_execution.yaml
base_train.yaml			models				toy.yaml			wandb.yaml

Reply:
This snippet says to run
```
python train.py --config conf/experiments/any-task_relu.yml
```

I'll fix `yml` -> `yaml` and the ambiguity in the code line